message.py

```diff
@@ -20,11 +28,16 @@ def clear_message(self) -> None:
 def build_off_command(id: int) -> bytes:
     return b'0,%d,0,0;' % id
 
+
 def build_on_command(id: int, color: int) -> bytes:
```

Comment:
Not sure about the protocol, but add percent as an optional param to build_on, maybe?